lib/common/grpc-service.js

```diff
@@ -332,24 +364,20 @@ GrpcService.convertValue_ = function(value) {
     convertedValue = {
       structValue: GrpcService.objToStruct_(value)
     };
-  } else if (is.date(value)) {
```

Comment:
Re-read the thread comments above, are we sure we want to remove this? It appears it will make the timestamp formatting of the UI viewer awkward?

Also, should we be converting this value to a Date object when decoding?